ens-improvement-proposals/ensip-11-evmchain-address-resolution.md

```diff
@@ -0,0 +1,90 @@
+---
+description: >-
+  Introduces coinType for EVM compatible chains (amending ENSIP9).
+---
+
+# ENSIP-9: EVM compatible Chain Address Resolution
+
+| **Author**    | Makoto Inoue \<makoto@ens.domains> |
+| ------------- | -------------------------------- |
+| **Status**    | Draft                            |
+| **Submitted** | 2022-01-13                       |
+
+### Motivation
```

Comment:
This needs an abstract describing the change at a high level.

Reply:
Added ` This specification standardises a way to derive coinType based on [Chain ID](https://chainlist.org)`

Reply:
The abstract should be a separate section, with a high-level description of the standard - reading it should be enough to get an idea for how it works.

Reply:
I see. Abstract added. BTW, ENSIP 9 which I copied as a template also lacks the abstract.